subprojects/core/src/main/java/org/gradle/api/internal/initialization/DefaultBuildLogicBuildQueue.java

```diff
@@ -96,4 +110,10 @@ private FileLock lockBuildLogicQueueFile() {
             "build logic queue"
         );
     }
+
+    private static List<TaskIdentifier.TaskBasedTaskIdentifier> withoutExecuted(List<TaskIdentifier.TaskBasedTaskIdentifier> tasks) {
```

Comment:
💅 It's a bit hard to process the name mentally, it fit the best as a name for extension function, like
`tasks.withoutExecuted()`. But given that we are in Java world, I'd prefer to avoid this "reverse" nature of the name with something more straight like `getTasksToExecute(tasks)`

Reply:
> It's a bit hard to process the name mentally,

How does it look from the call site for you:
```java
remaining = withoutExecuted(tasks);
```

Reply:
Current version looks good to me 👍